Lib/fontTools/cffLib/cff2_merge_funcs.py

```diff
@@ -0,0 +1,269 @@
+						b"Warning: {key} in default font Private dict is "
+						b"missing from another font, and was "
+						b"discarded.".format(key=key))
+					continue
+				try:
+					values = zip(*values)
+				except IndexError:
+					raise MergeDictError(key, value, values)
+				"""
+				Row 0 contains the first  value from each master.
+				Convert each row from absolute values to relative
+				values from the previous row.
+				e.g for three masters,	a list of values was:
+				master 0 OtherBlues = [-217,-205]
+				master 1 OtherBlues = [-234,-222]
+				master 1 OtherBlues = [-188,-176]
+				The call to zip() converts this to:
+				[(-217, -234, -188), (-205, -222, -176)]
+				and is converted finally to:
+				OtherBlues = [[-217, 17.0, 46.0], [-205, 0.0, 0.0]]
+				"""
+				dataList = []
+				prev_val_list = [0] * num_masters
+				any_points_differ = False
+				for val_list in values:
+					rel_list = [(val - prev_val_list[i]) for (
+							i, val) in enumerate(val_list)]
+					if (not any_points_differ) and pointsDiffer(rel_list):
+						any_points_differ = True
+					prev_val_list = val_list
+					deltas = var_model.getDeltas(rel_list)
+					# Convert numbers with no decimal part to an int.
+					deltas = [conv_to_int(delta) for delta in deltas]
+					# For PrivateDict BlueValues, the default font
+					# values are absolute, not relative to the prior value.
+					deltas[0] = val_list[0]
+					dataList.append(deltas)
+				# If there are no blend values,then
+				# we can collapse the blend lists.
+				if not any_points_differ:
+					dataList = [data[0] for data in dataList]
+			else:
+				values = [pd.rawDict[key] for pd in pds]
+				if pointsDiffer(values):
+					dataList = var_model.getDeltas(values)
+				else:
+					dataList = values[0]
+			private_dict.rawDict[key] = dataList
+
+
+class MergeCharError(TypeError):
+	def __init__(self, glyph_name, mergeError):
+		self.error_msg = "{mergeError} in glyph {glyph_name}".format(
+				mergeError=mergeError.error_msg,
+				glyph_name=glyph_name)
+		super(MergeCharError, self).__init__(self.error_msg)
+
+
+def merge_region_fonts(varFont, model, ordered_fonts_list, glyphOrder):
+	topDict = varFont['CFF2'].cff.topDictIndex[0]
+	default_charstrings = topDict.CharStrings
+	region_fonts = ordered_fonts_list[1:]
+	region_top_dicts = [
+			ttFont['CFF '].cff.topDictIndex[0] for ttFont in region_fonts
+				]
+	num_masters = len(model.mapping)
+	merge_PrivateDicts(topDict, region_top_dicts, num_masters, model)
+	merge_charstrings(default_charstrings,
+					  glyphOrder,
+					  num_masters,
+					  region_top_dicts, model)
+
+
+def merge_charstrings(default_charstrings,
+					  glyphOrder,
+					  num_masters,
+					  region_top_dicts,
+					  var_model):
+	for gname in glyphOrder:
+		default_charstring = default_charstrings[gname]
+		var_pen = CFF2CharStringMergePen([], num_masters, master_idx=0)
+		default_charstring.draw(var_pen)
+		for region_idx, region_td in enumerate(region_top_dicts):
+			region_idx += 1
```

Comment:
no need for this, `enumerate` can take a start value https://docs.python.org/3/library/functions.html#enumerate